tests/DocumentationExamplesTest.php

```diff
@@ -1540,6 +1540,85 @@ public function testCausalConsistency()
         ob_end_clean();
     }
 
+    /**
+     * @doesNotPerformAssertions
+     */
+    public function testWithTransactionExample()
+    {
+        $this->skipIfTransactionsAreNotSupported();
+
+        $uriString = static::getUri(true);
+
+        // phpcs:disable SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly
+        // Start Transactions withTxn API Example 1
+        /*
+         * For a replica set, include the replica set name and a seedlist of the members in the URI string; e.g.
+         * uriString = 'mongodb://mongodb0.example.com:27017,mongodb1.example.com:27017/admin?replicaSet=myRepl'
+         * For a sharded cluster, connect to the mongos instances; e.g.
+         * uriString = 'mongodb://mongos0.example.com:27017,mongos1.example.com:27017/admin'
```

Comment:
`/admin` looks redundant in both examples. I left a note about this in [DRIVERS-655](https://jira.mongodb.org/browse/DRIVERS-655) for @kay-kim, so maybe we can get that removed. If not, I suppose this will need to remain as-is for consistency.

Reply:
[DRIVERS-655](https://jira.mongodb.org/browse/DRIVERS-655) was updated and no longer includes "/admin".